wp-content/plugins/core/src/Blocks/Fields/Traits/With_Cta_Field.php

```diff
@@ -0,0 +1,63 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Project\Blocks\Fields\Traits;
+
+use Tribe\Libs\ACF\Field;
+use Tribe\Libs\ACF\Field_Group;
+
+trait With_Cta_Field {
```

Comment:
Should probably add some comments here about what this does, where it should be used.